dts/arm/cypress/psoc6_cm0.dtsi

```diff
@@ -15,6 +15,307 @@
 
 		/delete-node/ cpu@1;
 	};
+
+	soc {
+		intmux: intmux@40210020 {
+			/* see cypress,psoc6-int-mux.yaml */
+			compatible = "cypress,psoc6-intmux";
+			reg = <0x40210020 0x20>;
+			label = "INTMUX";
+			status = "disabled";
+			#address-cells = <1>;
+			#size-cells = <1>;
```

Comment:
technically we should have:

`ranges = <0x0 0x40210020 0x20>;`

(this will change the reg value to be the correct MMIO address, I don't think it impacts anything as it doesn't look like you are using the reg addr).

Reply:
Done!